go-controller/cmd/ovn-kube-util/app/ovn-db-exporter.go

```diff
@@ -391,17 +413,14 @@ var OvnDBExporterCommand = cli.Command{
 
 		// functions responsible for collecting the values and updating the prometheus metrics
 		go func() {
-			for {
-				ovnDBStatusMetricsUpdater("nb", "OVN_Northbound")
-				ovnDBStatusMetricsUpdater("sb", "OVN_Southbound")
-				time.Sleep(30 * time.Second)
+			dirDbMap := map[string]string{
```

Comment:
@girishmg should this be a const?

Reply:
@dcbw can you please explain what should be const here? Thanks.

Reply:
@girishmg nevermind, Go doesn't allow constant maps.